components/input-validation-mgt/org.wso2.carbon.identity.input.validation.mgt/src/main/java/org/wso2/carbon/identity/input/validation/mgt/utils/Constants.java

```diff
@@ -62,10 +62,11 @@ public static class Configs {
         public static final String JAVA_REGEX_PATTERN = "^((?=.*\\d)(?=.*[a-z])(?=.*[A-Z])).{8,100}$";
         public static final String DEFAULT_ALPHANUMERIC_REGEX_PATTERN = "^[a-zA-Z0-9]*$";
         public static final String DEFAULT_EMAIL_JAVA_REGEX_PATTERN =
-                "^[\\u00C0-\\u00FF\\w&&[^.+\\-_]](?:(?![.+\\-_]{2})[\\u00C0-\\u00FF\\w.+\\-]){0,63}(?<![+.\\-_])@" +
-                        "(?![+.\\-_])[\\w.+\\-]+\\.[a-zA-Z]{2,10}";
+            "(^[\\u00C0-\\u00FFa-zA-Z0-9](?:(?![.+\\-_]{2})[\\u00C0-\\u00FF\\w.+\\-]){0,63}(?=[\\u00C0-\\u00FFa-zA-Z0" +
```

Comment:
Any reason for updating these values?

Reply:
Previously we have configured default values of the PRIMARY userstore. Update them with secondary userstore values.